### DISCUSSION TITLE
Allow prompt editing without creating new version (draft mode)

### DISCUSSION THREAD

**Discussion by dveyarangi** [langfuse/langfuse#3893] [Ideas]
### Describe the feature or potential improvement

- allow to edit last, non production version of prompt. Maybe it is just me, but I often get ideas for improvements just after I click "save", and then i have to create another version.
And some wishful thinking - allow to compare prompts side by side, edited one with latest or another version

**Comment by marcklingen** [comment]
Thanks for the feedback! Agree that some sort of "draft mode" would be helpful to save changes without publishing a new version every time.

There is a thread for a potential diff view already: https://github.com/orgs/langfuse/discussions/1105
